src/plugins/intel_gpu/tests/unit/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1680,6 +1681,76 @@ TEST(activation_i32_fw_gpu, basic_yxfb_i32_funcs) {
             case activation_func::floor:
                 ASSERT_EQ((int32_t)std::floor(input_ptr[i]), output_ptr[i]);
                 break;
+            case activation_func::abs:
+                ASSERT_EQ(std::abs(static_cast<int32_t>(input_ptr[i])), output_ptr[i]);
+                break;
+            default:
+                break;
+            }
+        }
+    }
+}
+
+TEST(activation_i32_fw_gpu, basic_yxfb_u8_funcs) {
```

Comment:
can you merge this function with `TEST(activation_i32_fw_gpu, basic_yxfb_i32_funcs)`?
This looks like a copy-and-paste with minor change.

Reply:
Remove 'basic_yxfb_u8_funcs' tests